src/test/pages/admin/userManagement/userPage.ts

```diff
@@ -10,30 +11,351 @@ export default class AdminMenuPage {
+        await expect(this.titlePage).toBeVisible();
+        await expect(this.usernameLabel).toBeVisible();
+        await expect(this.usernameFieldSearch).toBeEditable();
+        await expect(this.userRoleLabel).toBeVisible();
+        await expect(this.userRole).toBeVisible();
+        await expect(this.employeeNameLabel).toBeVisible();
+        await expect(this.employeeName).toBeEditable()
+        await expect(this.statusLabel).toBeVisible();
+        await expect(this.status).toBeVisible();
+        await expect(this.resetBtn).toBeVisible();
+        await expect(this.searchBtn).toBeVisible();
+        await expect(this.addUserBtn).toBeVisible();
+        await expect(this.checkBox).toBeVisible();
+        await expect(this.usernameColumn).toBeVisible();
+        await expect(this.userRoleColumn).toBeVisible();
+        await expect(this.employeeNameColumn).toBeVisible();
+        await expect(this.statusColumn).toBeVisible();
+        await expect(this.actionColumn).toBeVisible();
+    }
+
+    async createEssUser() {
+        //employee: string, username: string, pass: string, confirm: string
+        await this.adminMenu.click();
+        await this.addBtn.click();
+        await this.userRole.click();
+        await this.userRoleOption.click();
+        await this.status.click();
+        await this.statusOption.click();
+        await this.employeeName.fill("t");
+        await this.employeeOption.click();
+        await this.usernameField.fill("usernamenttheu");
+        await this.passwordField.fill("admin123");
+        await this.confirmPassword.fill("admin123");
+        await this.submitBtn.click();
+    }
+
+    async afterCreateEssUser() {
+        await this.successToast.waitFor({ state: 'visible', timeout: 10000 });
+        await expect(this.newEssUser).toBeVisible();
+    }
+
+    async createAdminUser() {
+        //employee: string, username: string, pass: string, confirm: string
         await this.adminMenu.click();
         await this.addBtn.click();
         await this.userRole.click();
+        await this.adminRoleOption.click();
+        await this.status.click();
+        await this.statusOption.click();
+        await this.employeeName.fill("t");
+        await this.employeeOption.click();
+        await this.usernameField.fill("usernamenttheuAdmin");
+        await this.passwordField.fill("admin123");
+        await this.confirmPassword.fill("admin123");
+        await this.submitBtn.click();
+    }
+
+    async afterCreateAdminUser() {
+        await this.successToast.waitFor({ state: 'visible', timeout: 10000 });
+        await expect(this.newAdminUser).toBeVisible();
+    }
+
+    async searchUserName() {
+        await this.adminMenu.click();
+        await this.usernameFieldSearch.fill("usernamenttheu");
+        await this.searchBtn.click();
+    }
+
+    async afterSearchUserName() {
+        try {
+            const response = await axios.get(`${process.env.SEARCH_URL}`);
```

Comment:
Please use Playwright request instead of axios

Reply:
Done